lib/EntryManager/EntrySelector.js

```diff
@@ -4,6 +4,7 @@ import _ from 'lodash';
 import Switch from 'react-router-dom/Switch';
 import Route from 'react-router-dom/Route';
 import { FormattedMessage } from 'react-intl';
+import SafeHTMLMessage from '@folio/react-intl-safe-html';
```

Comment:
Please group that import to the `IfPermission` above. I see that folio imports stay in its own group.